### ISSUE TITLE
(M1) one of the sub-processes failed: com.docker.driver.amd64-linux -addr fd:3 -debug

### ISSUE THREAD

**Issue by arielweinberger** [docker/for-mac#5345]
- [ ] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID: Diagnostic stalling
Using Tech Preview

### Expected behavior
Docker Tech Preview starts and runs normally.

### Actual behavior
The following error shows up:
``` one of the sub-processes failed: com.docker.driver.amd64-linux -addr fd:3 -debug```

Diagnostics run forever.

### Information

  - macOS Version: Big Sur 11.1
  - Docker Desktop Version: Tech Preview v7

I was able to use the Tech Preview of Docker for about 3 weeks on my M1 Macbook Pro, and then suddenly the issue started occurring. Restarting, reinstalling etc does not help. I see others are experiencing the same.

I am not able to bring up the UI at all, it completely crashes after the erro comes up.

### Steps to reproduce the behavior

  1. Download Tech Preview version 7
  2. Run Docker
  3. Error shows up

![image](https://user-images.githubusercontent.com/4976416/107874963-7bcf7580-6ebd-11eb-8dc0-d0e3f8e0e13d.png)

**Comment by arielweinberger** [comment]
I have created an issue several days ago ([link](https://github.com/docker/for-mac/issues/5343)) and @StefanScherer replied with a potential solution that did not fix the issue.

I really appreciate what you do, and appreciate that you provided me with a potential solution, but wouldn't it be better to wait for my feedback about the solution before closing the support issue?

The issue still persists and because the previous one was closed, I had to create a new one.

Again, I appreciate what you do. But since the issue still persists I re-created it.
